app/src/main/java/org/thoughtcrime/securesms/loki/protocol/ClosedGroupsProtocolV2.kt

```diff
@@ -3,6 +3,7 @@ package org.thoughtcrime.securesms.loki.protocol
 import android.content.Context
```

Comment:
I couldn't find the bit where if the admin receives a `MEMBERS_ADDED` they send the latest encryption key pair to the added users. Can you point me to where that happens @hjubb?